pkg/knuu/instance_helper.go

```diff
@@ -124,6 +124,23 @@ func (i *Instance) deployPod() error {
 		return fmt.Errorf("failed to get image name: %v", err)
 	}
 
+	serviceAccountName := i.k8sName
```

Comment:
sa = podName, right?

Reply:
Yes, as each Pod will get its own SA, I will use the StatefulSet name as the SA name.